project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject puppetlabs/lein-ezbake "1.4.1-SNAPSHOT"
+(defproject puppetlabs/lein-ezbake "1.5.0-SNAPSHOT"
```

Comment:
And the job we ran: https://jenkins-enterprise.delivery.puppetlabs.net/view/All/job/enterprise_ezbake_release-clj_master/4/